config/environments/development.rb

```diff
@@ -31,9 +31,23 @@
   config.active_storage.service = :google
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.delivery_method = :smtp
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    :address              => "smtp.gmail.com",
+    :port                 => 587,
+    :user_name            => "nflkiddo.productions@gmail.com",
+    :password             => "xqtuktmwhlqjejlw",
```

Comment:
Oh no. You shouldn't commit this.  You should git reset back to before this commit and force push over. Instead, put this into an environment variable and load it in here.